renku/cli/dataset.py

```diff
@@ -447,6 +448,54 @@ def remove(client, names):
     click.secho('OK', fg='green')
 
 
+@dataset.command('export')
+@click.argument('id')
+@click.argument('provider')
+@click.option(
+    '-p',
+    '--publish',
+    is_flag=True,
+    help='Automatically publish exported dataset.'
+)
+@pass_local_client(clean=True, commit=True)
+def export_(client, id, provider, publish):
+    """Export data to 3rd party provider."""
+    secret_url = 'https://zenodo.org/account/settings/applications/tokens/new/'
+    config_key_secret = 'secret'
+
+    provider_id = provider
+    try:
+        provider = ProviderFactory.from_id(provider_id)
+    except KeyError:
+        raise BadParameter('Unknown provider.')
+
+    secret = client.get_value(provider_id, config_key_secret)
+
+    if secret is None:
+        text_prompt = WARNING + 'Secret not found!\n'
+        text_prompt += 'Create one at: {0}\n'.format(secret_url)
+        text_prompt += 'Auth secret:'
```

Comment:
```suggestion
        text_prompt += 'Auth secret'
```
`click.prompt` appends the `:`